vendor.mod

```diff
@@ -62,6 +62,7 @@ require (
 	github.com/miekg/pkcs11 v1.1.1 // indirect
 	github.com/moby/sys/symlink v0.2.0 // indirect
 	github.com/opencontainers/runc v1.1.7 // indirect
+	github.com/opencontainers/runtime-spec v1.1.0-rc.2 // indirect
```

Comment:
FWIW I was also updating the docker/docker dependency (https://github.com/docker/cli/pull/4341), and didn't get this indirect. I was curious why, but looks like https://github.com/moby/moby/commit/d222bf097cd4c8121900c292f0f6ff243773c3c5 (https://github.com/moby/moby/pull/43980) removes this dependency from `api/types`

Reply:
This PR no longer touches `vendor.mod`